pepys_import/file/importer.py

```diff
@@ -52,7 +52,7 @@ def can_load_this_file(self, file_contents) -> bool:
         :rtype: bool
         """
 
-    def core_load_this_file(self, data_store, path, file_object, datafile):
+    def load_this_file(self, data_store, path, file_object, datafile):
```

Comment:
@robintw - I've switched to this strategy - _slightly_ extending my Python skillset. 

But, I don't know if I'm correctly interpreting:
> The underscore prefix is meant as a hint to another programmer that a variable or method starting with a single underscore is intended for internal use. 

What do you think?

Reply:
I like it - and wish I'd thought of it! Underscore methods in Python are used quite frequently to show that it's for internal use only - and although that's not enforced, it is well-accepted convention (and things like IPython won't show auto-completion for methods that start with an underscore). I think this is a far nicer way of doing it than my original way - thanks.

Reply:
The, erm, tests were failing for my fix.  I think I've found the error, and am re-testing.

```
>       self._load_this_file(self, data_store, path, file_object, datafile)
E       TypeError: _load_this_file() takes 5 positional arguments but 6 were given
```
I _guess_ that another `self` was being prepended to the arguments in the call.

Reply:
Yes you're right: when you call a method as `self.blah(a, b, c)` then `self` is automatically injected into the argument list and doesn't need to be specified explicitly.